test/parallel/test-tls-basic-validations.js

```diff
@@ -135,3 +135,12 @@ assert.throws(() => { tls.createSecureContext({ maxVersion: 'fhqwhgads' }); },
                 code: 'ERR_TLS_INVALID_PROTOCOL_VERSION',
                 name: 'TypeError'
               });
+
+[undefined, null, 1, true].forEach((value) => {
+  assert.throws(() => {
+    tls.connect({ checkServerIdentity: value });
+  }, {
+    code: 'ERR_INVALID_ARG_TYPE',
+    name: 'TypeError',
+  });
+});
```

Comment:
```suggestion
for (const checkServerIdentity of [undefined, null, 1, true]) {
  assert.throws(() => {
    tls.connect({ checkServerIdentity });
  }, {
    code: 'ERR_INVALID_ARG_TYPE',
    name: 'TypeError',
  });
}
```

Reply:
Use `for of` instead of `forEach` as you suggested.